### PR TITLE
Interactive simulator patches

### PR DESCRIPTION
**Pull request by wackerow**
## Description
- Updates some copy around ether/ETH
- Updates `NotificationPopover` (previously named DemoOnlyPopover) and further rolls out its usage with more helper text
- Switched final "get wallet" link to use relative path, and thus keep it as an "internal" link that will not open in a new tab

## Preview URL
- https://ethereumorgwebsitedev01-simpatches.gatsbyjs.io/en/wallets/

### CODE REVIEW

**Comment by konopkja** [comment]
@wackerow can we please in the first flow second step change headline from "They are free apps you can download" to "Wallets are free apps you can download..."
image

File: src/components/Simulator/NotificationPopover.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          bg="background.highlight"
+          px={4}
+          py={2}
+          maxW="15rem"
+          borderRadius="base"
+          boxShadow="tooltip"
+          fontSize="xs"
+          {...restProps}
+        >
+          <Flex gap={2} zIndex={100000000}>
```
Comment (pettinarip) [inline src/components/Simulator/NotificationPopover.tsx:41]: Is this needed for a special case?

The Popover component already has the correct `zIndex` value https://chakra-ui.com/docs/styled-system/theme#z-index-values

```suggestion
          <Flex gap={2}>
```

**Review by pettinarip — APPROVED** [review]
LGTM!

File: src/components/Simulator/NotificationPopover.tsx (hunk 1 above)
Comment (corwintines) [reply src/components/Simulator/NotificationPopover.tsx:41]: I think we can get rid of this as well now that the Portal is added @wackerow

Comment (wackerow) [reply src/components/Simulator/NotificationPopover.tsx:41]: Yes... I meant to switch that to something more specific, but the Portal should fix this. Removing

**Review by corwintines — APPROVED** [review]

**Pull request merged**